### ISSUE TITLE
[Bug] Errors caught in sdk aren't being propagated to 

### ISSUE THREAD

**Issue by kaminskypavel** [mondaycom/monday-sdk-js#36]
**Describe the bug**
If you `monday.api ` throws an exception, its being watched somewhere in the parent window and is not 
propagated to the IFrame, therefor client apps cannot catch the errors

```
main-7200c6b71d99dbf4fa50.js:1 Error: Graphql validation errors
    at main-7200c6b71d99dbf4fa50.js:1
    at c (vendors-e5822d6167d2f886eaaa.js:2)
    at Generator._invoke (vendors-e5822d6167d2f886eaaa.js:2)
    at Generator.forEach.e.<computed> [as next] (vendors-e5822d6167d2f886eaaa.js:2)
    at o (vendors-e5822d6167d2f886eaaa.js:2)
    at s (vendors-e5822d6167d2f886eaaa.js:2)
```

the catcher (main-7200c6b71d99dbf4fa50) is hosted by monday
https://cdn7.monday.com/build/main-7200c6b71d99dbf4fa50.js

**Steps to reproduce**
1. Embed the SDK in an app
2. remove a permission (me:read)
2. Call `monday.api(...)` to fetch `{ me { name }}`
3. try to catch the error

**Comment by KODIKAS-NL** [comment]
It's the same issue I already reported two months ago, unfortunately no response from monday yet 😕
https://github.com/mondaycom/monday-sdk-js/issues/26

**Comment by PepperAddict** [comment]
Yes, please! 
I spent an embarrassing amount of time trying to figure out the error: https://community.monday.com/t/monday-api-sdk-not-authenticated-or-graphql-validation-errors/9633 🤦‍♀️

**Comment by vladmonday** [comment]
Hey @kaminskypavel, @KODIKAS-NL , @PepperAddict 
The issue was solved, the new version of SDK (0.1.1) is catching an error post message from Monday and rejects the promise.
If you will update your dependency to the new version it should work for you.
Thanks!

![image](https://user-images.githubusercontent.com/62242004/93332083-f2dae200-f829-11ea-9c42-0b39344c5ac7.png)
